src/realm/util/encryption_key.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef REALM_UTIL_ENCRYPTION_KEY_HPP
+#define REALM_UTIL_ENCRYPTION_KEY_HPP
+
+#include <realm/util/sensitive_buffer.hpp>
+
+#include <array>
+
+namespace realm::util {
+
+typedef SensitiveBuffer<std::array<uint8_t, 64>> EncryptionKey;
+
+typedef EncryptionKey::element_type EncryptionKeyStorageType;
+static_assert(std::is_pod<EncryptionKeyStorageType>::value); // for the use with the reinterpret_cast
```

Comment:
std::is_pod is deprecated and should not be used.

Reply:
Would is_standard_layout be more appropriate here?

Reply:
No, is_trivial. is_standard_layout is the part of is_pod that isn't applicable here (and is the reason the concept of pod types was removed from the standard).

Reply:
Thanks for the clarification.